jsonl_fstats/jsonl_fstats.py

```diff
@@ -142,9 +142,11 @@ def run():
             data = []
             for obj in valstats[key]:
                 data.append(valstats[key][obj])
-        npdata = np.asarray(data)
-        if data:
-            print(
+        if len(data) > 0:
+            npdata = np.asarray(data)
+        else:
+            npdata = np.asarray([' '])
```

Comment:
>>> np.mean(np.asarray([' ']))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bhering/.local/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2957, in mean
    out=out, **kwargs)
  File "/home/bhering/.local/lib/python3.6/site-packages/numpy/core/_methods.py", line 70, in _mean
    ret = umr_sum(arr, axis, dtype, out, keepdims)
TypeError: cannot perform reduce with flexible type